controllers/tools/views/user-dashboard.njk

```diff
@@ -0,0 +1,202 @@
+{% extends "layouts/main.njk" %}
+{% from "components/breadcrumb-trail/macro.njk" import breadcrumbTrail %}
+{% from "components/staff-status/macro.njk" import staffStatus with context %}
+
+{% block extraHead %}
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
```

Comment:
Any way we can avoid using bootstrap in _addition_ to our own styles? Cool with coming back to this to allow us to get something workable out, but good to avoid it if we can.

Reply:
Think for now it'd be more effort than it's worth (until we know how this tool gets used) but agree it'd be nice to avoid it if we can.